pipelines/preprocess_raw_data/create_cron_job.sh

```
@@ -1,4 +1,4 @@
-# This bash script will create a cron job that runs the sbatch submit_job.sh every 6 hours.
+# This bash script creates a cron job that runs sbatch submit_job.sh every 4 hours.
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)